core/src/modules/DynamicsModule/MEVPDynamics.cpp

```diff
@@ -86,23 +86,23 @@ void MEVPDynamics::update(const TimestepTime& tst)
     std::cout << tst.start << std::endl;
 
     // set the updated ice thickness and concentration
-    kernel.setData(hiceName, hice.data());
-    kernel.setData(ciceName, cice.data());
+    kernel.setData(hiceName, hice);
+    kernel.setData(ciceName, cice);
```

Comment:
Why `kernel.setData` and `getData` still here?

Reply:
The other dynamics implementations were forgotten. Updated to the new system.